staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -609,6 +621,10 @@ type VGPUDisplayOptions struct {
 	RamFB *FeatureState `json:"ramFB,omitempty"`
 }
 
+type PanicDevice struct {
+	Model *string `json:"model,omitempty"`
```

Comment:
```suggestion
	Model *PanicDeviceModel `json:"model,omitempty"`
```

Reply:
I also assume we don't want to support `address`?

https://libvirt.org/formatdomain.html#panic-device

Reply:
If needed, this support can be added later. But I dont see an usecase for it to justify carrying it around. This is only valid for `isa` and `pvpanic` models. And even there they use a default address if none is provided